### DISCUSSION TITLE
problem running meep script

### DISCUSSION THREAD

**Discussion by yafimvar** [NanoComp/meep#2443] [Q&A]
Hello , i am trying to run a meep script  on ubuntu VM virtual box.
I am using two files , an SH type file which run the py type file using terminal of the folder where they both located at the same folder. the SH file was creted using the youtube manual shown bellow.
https://www.youtube.com/watch?v=PIR2TeGOOMU&list=PLXB-QP1eJjhK7sH6zpWKLidE-VmSxcqlF&index=2
i got errors shown bellow.
Where did i go wrong?
Thanks.

first file:
https://drive.google.com/file/d/1vfk34AmtUVd4Dox2VNMnOHI-QYqcIPqT/view?usp=sharing
second file:
https://drive.google.com/file/d/1mZhQ5GY0KQmOnUv6O9I18f9cyKKfybgD/view?usp=sharing

john@john-VirtualBox:~/Documents$ bash run_python_meeddev.sh

EnvironmentNameNotFound: Could not find conda environment: meepdev
You can list all discoverable environments with `conda info --envs`.

Traceback (most recent call last):
  File "/home/john/Documents/test.py", line 1, in <module>
    import meep as mp
ModuleNotFoundError: No module named 'meep'
run_python_meeddev.sh: line 13: cd: test-out: No such file or directory
HDF5-DIAG: Error detected in HDF5 (1.10.7) thread 1:
  #000: ../../../src/H5F.c line 429 in H5Fopen(): unable to open file
    major: File accessibility
    minor: Unable to open file
  #001: ../../../src/H5Fint.c line 1644 in H5F_open(): unable to open file: time = Tue Mar 14 22:39:18 2023
, name = 'test-ez.h5', tent_flags = 0
    major: File accessibility
    minor: Unable to open file
  #002: ../../../src/H5FD.c line 741 in H5FD_open(): open failed
    major: Virtual File Layer
    minor: Unable to initialize object
  #003: ../../../src/H5FDsec2.c line 360 in H5FD__sec2_open(): unable to open file: name = 'test-ez.h5', errno = 2, error message = 'No such file or directory', flags = 0, o_flags = 0
    major: File accessibility
    minor: Unable to open file
h5topng error: error opening HD5 file
run_python_meeddev.sh: line 16: convert: command not found
rm: cannot remove 'test-ez*.t*.png': No such file or directory
run_python_meeddev.sh: line 18: cd..: command not found

**Comment by smartalecH** [comment]
As the top of your error stack says:

```
EnvironmentNameNotFound: Could not find conda environment: meepdev
```

Your conda env is not set up properly.

**Reply by yafimvar** [reply]
Hello, i googled meepdev, there is no such environment.
is there some alternative manual (video is best) which i could try to run the example script?
Thanks.
